### DISCUSSION TITLE
Error message on pages .../shortcodes.php on line 40

### DISCUSSION THREAD

**Discussion by VenturaCntyCentralOffice** [code4recovery/12-step-meeting-list#1300] [Q&A] [answered]
We are receiving this error message on our pages, it is not browser specific.

Warning: Undefined array key "location" in /home/aaventura/www/www/wp-content/plugins/12-step-meeting-list/includes/shortcodes.php on line 40

Warning: Undefined array key "location" in /home/aaventura/www/www/wp-content/plugins/12-step-meeting-list/includes/shortcodes.php on line 40

**Comment by joshreisner** [comment, marked as answer]
hm, sorry about that! we'll fix it so it doesn't happen. in the meantime if you wanted to fix it locally we typically suggest that people use a location name, so editing these online meetings to have a location name, eg "Online Meeting" would make it go away.

<img width="347" alt="Screenshot 2024-01-11 at 11 23 18 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/73855865-7cfa-42ef-906e-004fa8ba0051">

**Reply by joshreisner** [reply]
i see the issue has gone away for now, but would likely recur the next time there is a meeting without a location name in the next 5 upcoming meetings…

**Comment by VenturaCntyCentralOffice** [comment]
Thanks Josh, I had noticed it went away. Next time I see it I will check the meeting in the upcoming list and check them.
Matthew
    On Thursday, January 11, 2024 at 12:53:49 PM PST, Josh Reisner ***@***.***> wrote:  
 
 


i see the issue has gone away for now, but would likely recur the next time there is a meeting without a location name in the next 5 upcoming meetings…
